shared/version/api.go

```diff
@@ -272,6 +272,7 @@ var APIExtensions = []string{
 	"clustering_join_token",
 	"clustering_description",
 	"server_trusted_proxy",
+	"clustering_update_certs",
```

Comment:
This change should go into a `api: clustering_update_certs` commit along with the matching change to `doc/api-extensions.md`

Reply:
Looks like the `doc/api-extensions.md` part is actually missing, so that needs to be added as part of this.

Reply:
I think it would be a good idea to also add another commit `doc/cluster: Add section on certificate update` and add a new section to doc/cluster.md which says something along the lines of:

```
## Updating the cluster certificate
In a LXD cluster, all servers respond with the same shared certificate. This is usually a standard self-signed certificate with an expiry set to 10 years.

If you wish to replace it with something else, for example a valid certificate obtained through Let's Encrypt, `lxc cluster update-certificate` can be used to replace the certificate on all servers in your cluster.
```